### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by 7iomka** [travisluong/shadriz#4]
<img width="954" alt="image" src="https://github.com/user-attachments/assets/7d0848e1-5911-4bb1-91c7-202f27e60263">

Mac os, node 20.x
Similar results with pnpm / npm

**Comment by travisluong** [comment]
just released v5.0.0. let me know if you're still seeing that issue with the new version.

**Comment by 7iomka** [comment]
<img width="753" alt="image" src="https://github.com/user-attachments/assets/4ae43565-8d49-473f-9d6c-16a8f0e34f8e">


Just waiting for something without any response.
I think shadcn asks some questions for init if not see any files inside directory, but with your command we can't make response for it and don't see as well.

**Comment by travisluong** [comment]
yes, i think there is an issue with shadcn initiating some interactive prompt. i believe it is related to this: https://ui.shadcn.com/docs/react-19

i'm looking for a way to work around this now.

**Comment by travisluong** [comment]
this seems to be a known issue on the shadcn repo: https://github.com/shadcn-ui/ui/issues/5557

**Comment by travisluong** [comment]
@7iomka just pushed a fix in v5.0.1. the interactive prompts from shadcn should work now. you'll have to pick either `--force` or `--legacy-peer-deps`.

**Comment by travisluong** [comment]
if there are still issues, `npm config set legacy-peer-deps true` seems to help.

**Comment by travisluong** [comment]
@7iomka i think i see what the issue is. you have to first create the next app:

```
npx create-next-app@latest my-app --typescript --eslint --tailwind --app --no-src-dir --no-import-alias --turbopack
```

second, you have to cd into the directory and run init

```
cd my-app
npx shadts@latest init
```

i think you may have ran init without first creating the app.
